src/Mintette/MintetteOptions.hs

```diff
@@ -34,6 +37,13 @@ data ServeOptions = ServeOptions
     , cloActionLogsLimit :: Word
     }
 
+data AddToBankOptions = AddToBankOptions
+    { cloMintetteHost     :: String
```

Comment:
We have a convention that record fields names start with prefix which is either lower-cased name of type (for types with short names like `Coin`) or abbreviated name of type (e. g. `NewPeriodData` -> `npdMintettes`). `Options` is only one word and it's relatively long, so I used `clo` as abbreviation (which stands for `command line option`). Fields in `ServeOptions` also have this prefix, but that was by mistake (because of copy-paste). They should have `so` prefix instead of `clo`. And here you'd better use `atbo` prefix, that way there wouldn't be a duplicated `cloSecretKeyPath` name.